.husky/pre-push

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-prettier --check --ignore-unknown .
+npx prettier --check --ignore-unknown .
```

Comment:
c'est quoi l'intérêt de mettre npx ?

Reply:
Je n'ai pas installé `prettier` en global sur ma machine, donc ça me met une erreur, dans l'idée d'avoir un set-up sans trop de contrainte, je préfère éviter de forcer à installer des nodes modules en global.
Avec npx, ça va chercher le paquet sur npm à la volée avant d'exécuter, donc pas besoin d'installer quoi que ce soit sur la machine (à part node/npm)

Reply:
Par ailleurs, je me suis demandé à quoi sert ce prettier au pre-push, sachant que c'est déjà exécuter au pre-commit, ça fait pas un peu doublon ?

Reply:
non c'est un check pas un fix, ça permet de détecter quand on change des fichiers pas détecté par le lintstage (genre des règles etc)

Reply:
> Je n'ai pas installé `prettier` en global sur ma machine, donc ça me met une erreur, dans l'idée d'avoir un set-up sans trop de contrainte, je préfère éviter de forcer à installer des nodes modules en global. Avec npx, ça va chercher le paquet sur npm à la volée avant d'exécuter, donc pas besoin d'installer quoi que ce soit sur la machine (à part node/npm)

??? je pensais que ça l'executais à partir du nodes_modules il ne me semble pas avoir installé prettier en global non plus

Reply:
Bah essaye de lancer `prettier` en ligne de commande, si tu as le message `zsh: command not found: prettier` tu ne l'as pas installé, sinon `npm uninstall -g prettier`